lib/plugins/aws/utils/resolveCfImportValue.test.js

```diff
@@ -0,0 +1,36 @@
+'use strict';
+
+const expect = require('chai').expect;
+const sinon = require('sinon');
+const AwsProvider = require('../provider/awsProvider');
+const Serverless = require('../../../Serverless');
+const resolveCfImportValue = require('./resolveCfImportValue');
+
+describe('#resolveCfImportValue', () => {
+  let provider;
+  let providerStub;
+
+  beforeEach(() => {
+    const serverless = new Serverless();
+    provider = new AwsProvider(serverless, {});
+    providerStub = sinon.stub(provider, 'request');
```

Comment:
Let's not use `Serverless` and `AwsProvider`, there's a lot of machinery in them which may introduce side effects.

I would simply inject plain `provider` object with `request` method that returns pre-prepared responses.